include/util/graph_loader.hpp

```diff
@@ -50,26 +49,26 @@ NodeID loadNodesFromFile(storage::io::FileReader &file_reader,
     coordinates.resize(number_of_nodes);
     osm_node_ids.reserve(number_of_nodes);
 
-    extractor::ExternalMemoryNode current_node;
+    extractor::QueryNode current_node;
     for (NodeID i = 0; i < number_of_nodes; ++i)
```

Comment:
Hmm, it's somewhat unrelated, but I wonder if we could get some speedup here by reading blocks of `current_node` and looping over the blocks.  Reading these objects one at a time almost certainly is creating a lot of syscall overhead......